test/integration_test.js

```diff
@@ -3,6 +3,8 @@ const {
 } = require('../scalardl-node-client-sdk');
 const fs = require('fs');
 
+const cassandra = require('cassandra-driver');
+
```

Comment:
I don't get the rules of empty lines in this file. Can you clarify ?

Reply:
Owh, I'm sorry, the empty line on line 5 should be removed, and there are going to be empty lines in between the `require` part, and variable declaration part (`properties` and `clientService`) and the test part (`describe`).